src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -219,7 +219,8 @@ ShouldSyncTableMetadata(Oid relationId)
  * following queries:
  *
  * (i)   Query that populates pg_dist_node table
- * (ii)  Queries that create the clustered tables
+ * (ii)  Queries that create the clustered tables (including foreign keys,
```

Comment:
We don't use the term `clustered` anymore. But, let's not touch it in this commit.